airbyte-integrations/connectors/source-shopify/source_shopify/schemas/shop.json

```diff
@@ -0,0 +1,172 @@
+{
+  "type": "object",
+  "properties": {
+    "address1": {
+      "type": ["null", "string"]
+    },
+    "address2": {
+      "type": ["null", "string"]
+    },
+    "auto_configure_tax_inclusivity": {
+      "type": ["null", "string"]
+    },
+    "checkout_api_supported": {
+      "type": ["null", "boolean"]
+    },
+    "city": {
+      "type": ["null", "string"]
+    },
+    "country": {
+      "type": ["null", "string"]
+    },
+    "country_code": {
+      "type": ["null", "string"]
+    },
+    "country_name": {
+      "type": ["null", "string"]
+    },
+    "county_taxes": {
+      "type": ["null", "boolean"]
+    },
+    "created_at": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "customer_email": {
+      "type": ["null", "string"]
+    },
+    "currency": {
+      "type": ["null", "string"]
+    },
+    "domain": {
+      "type": ["null", "string"]
+    },
+    "enabled_presentment_currencies": {
+      "items": {
+        "type": ["null", "string"]
+      },
+      "type": ["null", "array"]
```

Comment:
The `type` of `array` should have the property of `items` with the nested data inside of array of data. Please make sure you have added that as well.

Reply:
I am not following what you mean here. The `items` property is defined above, the same way as it is for other resources, [for example](https://github.com/cobobrien/airbyte/blob/84aad359aa68808cee059b16e9a83138d7c72781/airbyte-integrations/connectors/source-shopify/source_shopify/schemas/price_rules.json#L22)

Reply:
Should it be different?

Reply:
Yes it is an array of strings